pkg/sysfs/system.go

```diff
@@ -941,6 +961,36 @@ func (sys *system) discoverPackages() error {
 	return nil
 }
 
+func (sys *system) discoverSst() error {
+	if !SstSupported() {
+		sys.Info("Speed Select Technology (SST) support not detected")
+		return nil
+	}
+
+	for _, pkg := range sys.packages {
+
```

Comment:
nitpick: is this blank line intentional ?

Reply:
Nope. Removed